source/server/spongineer/logs.rst

```diff
@@ -12,54 +12,48 @@ logfiles from SpongeForge and SpongeVanilla servers including short descriptions
 SpongeForge logfiles
 ====================
 
-SpongeForge writes several logfiles to the ``/logs`` folder located inside your servers directory. As of Forge 1521
-these are:
+SpongeForge writes several logfiles to the ``/logs`` folder located inside your servers directory. 
+As of Forge 1.12.2 - 14.23.4.2705 these are: 
 
-1. ``fml-junk-earlystartup.log``
-#. ``fml-server-latest.log``
+1. ``debug.log``
 #. ``latest.log``
 
-fml-junk-earlystartup.log
+debug.log
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. note::
 
  Only a few example lines are shown here. To read the full example log, follow this link:
- :download:`SpongeForge 1521 fml-junk-earlystartup.log file </files/logs/forge-1521-fml-junk-earlystartup.txt>`
-
-fml-server-latest.log
-~~~~~~~~~~~~~~~~~~~~~
-
-.. note::
-
- Only a few example lines are shown here. To read the full example log, follow this link:
- :download:`SpongeForge 1521 fml-server-latest.log </files/logs/forge-1521-fml-server-latest.txt>`
+ :download:`SpongeForge 1.12.2 - 7.1.0-BETA-3126 debug.log file 
+ </files/logs/SpongeForge_1.12.2-2705-7.1.0-BETA-3136_debug.log.txt>`
 
 .. code-block:: none
 
- [main/INFO] [FML/]: Forge Mod Loader version 11.14.3.1521 for Minecraft 1.8 loading
- [main/INFO] [FML/]: Java is Java HotSpot(TM) 64-Bit Server VM, version 1.8.0_51, running on Windows 8.1:amd64:6.3, installed at ##PATH_TO_JAVA_HERE##
- [main/DEBUG] [FML/]: Java classpath at launch is forge.jar
- [main/DEBUG] [FML/]: Java library path at launch is ##PATH_TO_JAVA_HERE##
+ [main/INFO] [FML]: Forge Mod Loader version 14.23.4.2705 for Minecraft 1.12.2 loading
+ [main/INFO] [FML]: Java is Java HotSpot(TM) 64-Bit Server VM, version 1.8.0_162, running on Windows 10:amd64:10.0, installed at C:\Program Files\Java\jre1.8.0_162
+ [main/DEBUG] [FML]: Java classpath at launch is:
+ [main/DEBUG] [FML]:     forge-1.12.2-14.23.4.2705-universal.jar
+ [main/DEBUG] [FML]: Java library path at launch is:
+ [main/DEBUG] [FML]:     C:\ProgramData\Oracle\Java\javapath
 
 The example log indicates that we're running:
 
-* Forge 11.14.3.1521 (Version 1521)
-* Java 8 64bit Update 51
-* Windows 8.1 x64
+* Forge 14.23.4.2705 (Version 2705)
+* Java 8 64bit Update 162
+* Windows 10 x64
 * the ``directory`` Java was installed to (Line 4)
 
 .. warning::
 
- SpongeForge won't run on Java 6 (``1.6.x``) or Java 7 (``1.7.x``). If you encounter an error stating that you run
- an older Java build than Java 8, please update your JRE to ``1.8.x`` and try again!
+ SpongeForge won't run anything other than Java 8, and only Update 20 or newer. If you encounter an error stating that 
+ you run an older or newer Java build than Java 8, please update your JRE to ``1.8.x`` and try again!
```

Comment:
> please update your

Maybe use change since it is not necessarily an update.

Reply:
not happy with these two sentences in general, will see tonight how to rewrite that

if someone has a good idea let me know